File: RepoTransBench/target_projects/Python/Java/mardix_flask-recaptcha/src/test/java/com/example/flaskrecaptcha/publictests/PublicFlaskRecaptchaTest.java
package com.example.flaskrecaptcha.publictests;

import com.example.flaskrecaptcha.ReCaptcha;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

public class PublicFlaskRecaptchaTest {

    @Test
    void testPublicSetAndGetSiteKey() {
        ReCaptcha recaptcha = new ReCaptcha();
        recaptcha.site_key = "different_public_key";
        assertEquals("different_public_key", recaptcha.site_key);
    }

    @Test
    void testPublicSetAndGetSecretKey() {
        ReCaptcha recaptcha = new ReCaptcha();
        recaptcha.secret_key = "different_public_secret";
        assertEquals("different_public_secret", recaptcha.secret_key);
    }

    @Test
    void testPublicLanguageSetterAndGetter() {
        ReCaptcha recaptcha = new ReCaptcha();
        recaptcha.language = "fr";
        assertEquals("fr", recaptcha.language);
    }

    @Test
    void testPublicThemeSetterAndGetter() {
        ReCaptcha recaptcha = new ReCaptcha();
        recaptcha.theme = "dark";
        assertEquals("dark", recaptcha.theme);
    }
}

File: RepoTransBench/target_projects/Python/Go/mardix_flask-recaptcha/public_tests/public_flask_recaptcha_test.go
package public_tests

import (
	"testing"
	"github.com/stretchr/testify/assert"
)

type ReCaptcha struct {
	SiteKey   string
	SecretKey string
	Language  string
	Theme     string
}

func TestPublicSetAndGetSiteKey(t *testing.T) {
	recaptcha := &ReCaptcha{}
	recaptcha.SiteKey = "different_public_key"
	assert.Equal(t, "different_public_key", recaptcha.SiteKey)
}

func TestPublicSetAndGetSecretKey(t *testing.T) {
	recaptcha := &ReCaptcha{}
	recaptcha.SecretKey = "different_public_secret"
	assert.Equal(t, "different_public_secret", recaptcha.SecretKey)
}

func TestPublicLanguageSetterAndGetter(t *testing.T) {
	recaptcha := &ReCaptcha{}
	recaptcha.Language = "fr"
	assert.Equal(t, "fr", recaptcha.Language)
}

func TestPublicThemeSetterAndGetter(t *testing.T) {
	recaptcha := &ReCaptcha{}
	recaptcha.Theme = "dark"
	assert.Equal(t, "dark", recaptcha.Theme)
}